event/feed.go

```diff
@@ -63,6 +70,10 @@ func (f *Feed) init(etype reflect.Type) {
 	f.sendLock = make(chan struct{}, 1)
 	f.sendLock <- struct{}{}
 	f.sendCases = caseList{{Chan: reflect.ValueOf(f.removeSub), Dir: reflect.SelectRecv}}
+	if metrics.Enabled {
+		f.meter = metrics.GetOrRegisterTimer("event/feed/send/"+
+			strings.ReplaceAll(etype.Name(), "*", "ptr_"), nil)
```

Comment:
Note: need to verify if this works for all Go type names that we may be sending over the channel, and what names the `metrics` package accepts as valid names.